src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -46,6 +46,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string SpellCheck = nameof(SpellCheck);
         public const string Suppression = nameof(Suppression);
         public const string AddOverloads = nameof(AddOverloads);
+        public const string AddShadows = nameof(AddShadows);
```

Comment:
If things are merged, do you need this anymore?

Reply:
Removed